JiraPS/Private/Resolve-ErrorWebResponse.ps1

```diff
@@ -44,7 +44,29 @@ function Resolve-ErrorWebResponse {
             try {
                 $responseObject = ConvertFrom-Json -InputObject $responseBody -ErrorAction Stop
 
+                $errorArray = @()
                 foreach ($_error in ($responseObject.errorMessages + $responseObject.errors)) {
+
+                    foreach ($_err in $_error.PSObject.Properties.Value) {
+                        if ($_err -like "*You must specify a summary of the issue." -Or $_err -like "*is required.") {
+                            Write-Debug $_err
+                            $errorArray += $_err
+                        } else {
+                            # unhandled value
+                        }
+                    }
+
+                    if ($errorArray.count -gt 0) {
+                        $errorParameter = @{
+                            ExceptionType = "System.Net.Http.HttpRequestException"
```

Comment:
Please use the same exception type as Invoke-WebRequest returns
(`System.Net.WebException` for PSVersion -lt 6 and `Microsoft.PowerShell.Commands.HttpResponseException` for PSVersion -ge 6)